templates/CRM/Contact/Form/Edit/Email.tpl

```diff
@@ -26,8 +26,8 @@
 
 <tr id="Email_Block_{$blockId}">
   <td>{$form.email.$blockId.email.html|crmAddClass:email}&nbsp;{$form.email.$blockId.location_type_id.html}
-    <div class="clear"></div>
-    {if $className eq 'CRM_Contact_Form_Contact'}
+    {if $className eq 'CRM_Contact_Form_Contact' and $form.email.$blockId.signature_html.html}
+      <div class="clear"></div>
```

Comment:
I moved the 'clear' inside here on purpose. It would be clutter otherwise (we're inside a table/td, risks of breaking themes are very low).

Reply:
If you have debugging turned on this adds a new E_NOTICE `Trying to access array offset on value of type null` because `$form.email.$blockId.signature_html.html` may not exist anymore. In this case it would be ok to use empty():

`{if $className eq 'CRM_Contact_Form_Contact' and !empty($form.email.$blockId.signature_html.html)}`